puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello.groovy

```diff
@@ -18,7 +14,7 @@ pipeline {
 
             steps {
 
-                mash("katello-mash-split-3.13.py")
+                mash("katello-mash-split.py", katello_version)
```

Comment:
doesn't this script also need to be modified?

Reply:
define modified, it has a branch map, so it needs a modification at branching time (which I did for 3.14), but I made it not need to copy the script around as we had before.

Reply:
ah, i missed the previous comment about making it accept a param.